typescript/src/index.ts

```diff
@@ -188,6 +189,24 @@ export interface TBTC {
     bitcoinClient: BitcoinClient
   ): Promise<void>
 
+  /**
+   * Prepares the proof of a redemption transaction and submits it to the
+   * Bridge on-chain contract.
+   * @param transactionHash - Hash of the transaction being proven.
+   * @param mainUtxo - Recent main UTXO of the wallet as currently known on-chain.
+   * @param walletPubKeyHash - 20-byte public key hash of the wallet
+   * @param bridge - Interface to the Bridge on-chain contract.
```

Comment:
`Interface to the Bridge on-chain contract.` -> `Handle to the Bridge on-chain contract.` and let's populate it to `redemption.ts`.